Source/Urho3D/Scene/Serializable.cpp

```diff
@@ -678,6 +678,44 @@ bool Serializable::LoadFile(const ea::string& resourceName)
     return Load(realResourceName);
 }
 
+bool Serializable::SaveFile(const ea::string& destinationPath)
+{
+    return SaveFile(destinationPath,"RootNode");
```

Comment:
Space after comma. Also as per our naming conventions root node should be called "root". Starts with lower-case and we should drop "node" because it may be confused with scene node.

Reply:
as discussed we will omit for now due to unexpected results